website/www/site/content/en/get-started/tour-of-beam.md

```diff
@@ -16,81 +16,6 @@ See the License for the specific language governing permissions and
-We introduce the built-in `ReadFromText` and `WriteToText` transforms.
-We also see how we can read from CSV files, read from a SQLite database, write fixed-sized batches of elements, and write windows of elements.
-
-{{< button-colab url="https://colab.research.google.com/github/apache/beam/blob/master/examples/notebooks/tour-of-beam/reading-and-writing-data.ipynb" >}}
-
-### Windowing
-
-In this notebook we go through how to aggregate data based on time intervals, or in streaming pipelines.
-We introduce the `GlobalWindow`, `FixedWindows`, `SlidingWindows`, and `Sessions`.
-
-{{< button-colab url="https://colab.research.google.com/github/apache/beam/blob/master/examples/notebooks/tour-of-beam/windowing.ipynb" >}}
-
-### DataFrames
-
-Beam DataFrames provide a pandas-like [DataFrame](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.html)
-API to declare Beam pipelines.
-To learn more about Beam DataFrames, take a look at the
-[Beam DataFrames overview](/documentation/dsls/dataframes/overview) page.
-
-{{< button-colab url="https://colab.research.google.com/github/apache/beam/blob/master/examples/notebooks/tour-of-beam/dataframes.ipynb" >}}
-
-## Transforms
-
-Check the [Python transform catalog](/documentation/transforms/python/overview/)
-for a complete list of the available transforms.
-
-### Element-wise transforms
-
-#### Map
-
-Applies a simple one-to-one mapping function over each element in the collection.
-
-{{< button-colab url="https://colab.research.google.com/github/apache/beam/blob/master/examples/notebooks/documentation/transforms/python/elementwise/map-py.ipynb" >}}
-
-#### FlatMap
-
-Applies a simple one-to-many mapping function over each element in the collection. The many elements are flattened into the resulting collection.
-
-{{< button-colab url="https://colab.research.google.com/github/apache/beam/blob/master/examples/notebooks/documentation/transforms/python/elementwise/flatmap-py.ipynb" >}}
-
-#### Filter
-
-Given a predicate, filter out all elements that don’t satisfy that predicate.
-
-{{< button-colab url="https://colab.research.google.com/github/apache/beam/blob/master/examples/notebooks/documentation/transforms/python/elementwise/filter-py.ipynb" >}}
-
-#### Partition
-
-Separates elements in a collection into multiple output collections.
-
-{{< button-colab url="https://colab.research.google.com/github/apache/beam/blob/master/examples/notebooks/documentation/transforms/python/elementwise/partition-py.ipynb" >}}
-
-#### ParDo
-
-A transform for generic parallel processing. It's recommended to use `Map`, `FlatMap`, `Filter` or other more specific transforms when possible.
-
-{{< button-colab url="https://colab.research.google.com/github/apache/beam/blob/master/examples/notebooks/documentation/transforms/python/elementwise/pardo-py.ipynb" >}}
+Please [click here](https://tour.beam.apache.org/) to access "the Tour of Beam".
```

Comment:
I think this would benefit from a (very brief) description of what this is. Maybe just something like "Tour of Beam is an interactive way of learning to write Beam code with a sandbox where you can write and run pipelines as you walk through various concepts. Please [click here](https://tour.beam.apache.org/) to try it out!"